postgrest.cabal

```diff
@@ -24,6 +24,7 @@ Flag CI
 executable postgrest
   main-is:             PostgREST/Main.hs
   default-extensions:  OverloadedStrings, ScopedTypeVariables, QuasiQuotes, LambdaCase
+  ghc-options:         -threaded -rtsopts -with-rtsopts=-N
```

Comment:
i saw some ghc options in the stack file also, not sure of how things work here (where is the place to put them) but maybe they should all be in one place